packages/backend-lalr/data/HappyTemplate.hs

```diff
@@ -1,67 +1,39 @@
 -- $Id: GenericTemplate.hs,v 1.26 2005/01/14 14:47:22 simonmar Exp $
 
-#ifdef HAPPY_GHC
-#  if !defined(__GLASGOW_HASKELL__)
-#    error `HAPPY_GHC` is defined but this code isn't being built with GHC.
-#  endif
-#  define ILIT(n) n#
-#  define IBOX(n) (Happy_GHC_Exts.I# (n))
-#  define FAST_INT Happy_GHC_Exts.Int#
+#if !defined(__GLASGOW_HASKELL__)
+#  error This code isn't being built with GHC.
+#endif
+#define ILIT(n) n#
+#define IBOX(n) (Happy_GHC_Exts.I# (n))
+#define FAST_INT Happy_GHC_Exts.Int#
```

Comment:
I think it would be good to manually inline these `#define`s.

Reply:
Done